routers/api/v1/api.go

```diff
@@ -994,6 +994,9 @@ func Routes() *web.Route {
 					m.Post("", reqToken(), reqRepoWriter(unit.TypeCode), bind(api.CreateTagOption{}), repo.CreateTag)
 					m.Delete("/*", reqToken(), repo.DeleteTag)
 				}, reqRepoReader(unit.TypeCode), context.ReferencesGitRepo(true))
+				m.Group("/actions", func() {
+					m.Get("/tasks", repo.ListActionTasks)
```

Comment:
Thank you very much for your contribution 🙏 To align with GitHub Actions so that other tools can read similar data from the API endpoints, can you adjust the URL to match the format `/repos/{owner}/{repo}/actions/runs` and the output schema to be something like (maybe not all the fields are needed yet):
```
{
  "total_count": 1,
  "workflow_runs": [
    {
      "id": 30433642,
      "name": "Build",
      "head_branch": "master",
      "head_sha": "acb5820ced9479c074f688cc328bf03f341a511d",
      "run_number": 562,
      "event": "push",
      "display_title": "Update README.md",
      "status": "queued",
      "conclusion": null,
      "workflow_id": 159038,
      "created_at": "2020-01-22T19:33:08Z",
      "updated_at": "2020-01-22T19:33:08Z",
      "actor": {
// ...
      },
      "run_started_at": "2020-01-22T19:33:08Z",
      "triggering_actor": {
     // ...
      },
     
    }
  ]
}
```

If you have any questions or aren't able to take on the above due to time constraints or other reason, please let me know and I can submit the changes.

Reply:
@chesteripz please resolve the change request.